ioservice/cob_foms.c

```diff
@@ -1139,6 +1151,10 @@ static int cd_cob_delete(struct m0_fom            *fom,
 	if (rc == 0)
 		M0_LOG(M0_DEBUG, "Cob deleted successfully.");
 
+	key.cbk_pfid = pver; 
+	key.cbk_user_id = M0_BYTECOUNT_USER_ID;
+	rc = cob_bytecount_decrement(cob, &key, cob_size, m0_fom_tx(fom));
```

Comment:
Move this code before cob delete, as domain in cob may not be available.

Reply:
Done.